definitions/variable/energy/energy-services.yaml

```
@@ -42,3 +42,12 @@
 - Energy Service|Transportation|Passenger|{Transport}:
     description: Provision of energy services related to passenger {Transport}
     unit: Gpkm/yr
+
+- Energy Service|Night Lights|Intensity|Minimum:
+    description:  The value ranges from 0 to 63 with higher value representing higher light intensity over 
+        the night. The input parameter sets the minimum light intensity value under which the model will consider a settlement as electrified.
+    unit: -
```

Similar comment to the one in energy-services.yaml, is this a model output or a model parameter? 
If it's the former totally fine to keep if it's the latter I don't think it's needed.
In any case for dimensionless quantities we just leave the unit blank"
```suggestion
    unit:
```